packages/common-common/tsconfig.json

```diff
@@ -1,4 +1,23 @@
 {
+  "compilerOptions": {
```

Comment:
We shouldn't need this since we're extending the root tsconfig.

Reply:
for rich

Reply:
yep, removed